Makefile

```diff
@@ -1,34 +1,37 @@
-DESTDIR=
-PREFIX=
-dirs = etc man usr
-files = Makefile README LICENSE setup.py cloud-regionsrv-client.spec
+buildroot = /
 
-nv = $(shell rpm -q --specfile --qf '%{NAME}-%{VERSION}|' *.spec | cut -d'|' -f1)
-verSpec = $(shell rpm -q --specfile --qf '%{VERSION}|' *.spec | cut -d'|' -f1)
-verSrc = $(shell cat lib/cloudregister/VERSION)
-ifneq "$(verSpec)" "$(verSrc)"
-    $(error "Version mismatch, will not take any action")
-endif
+python_version = 3
```

Comment:
We are going to have problems wit this at some point. The intention is to drop the `python3` symlink at some point, we should see if we cannot figure out some other way to find the appropriate Python interpreter.

Reply:
> The intention is to drop the python3 symlink at some point

I doubt this will ever happen. But if it happens it's easy to change python_lookup_name to watch for `python` and `python3` as long as just `python` is considered to be python v2.x the above is the most robust way to do it. I use this concept in kiwi and it was approved by all distros (Fedora, RHEL, Debian, Ubuntu) to be safe. If SUSE really makes that move I'm happy to follow and add code, until then there is no need to mess with it